runtime/buildtools.mk

```diff
@@ -97,7 +97,7 @@ TRACEGEN_DEFINITION_SENTINELS := $(patsubst %.tdf,%.tracesentinel,$(TRACEGEN_DEF
 
 # process TDF files to generate RAS tracing headers and C files
 trace_merge : buildtrace
-	@$(MAKE) -f buildtools.mk $(TRACEGEN_DEFINITION_SENTINELS)
+	@$(MAKE) -f buildtools.mk $(TRACEGEN_DEFINITION_SENTINELS) SPEC=$(SPEC)
```

Comment:
Please move `SPEC=$(SPEC)` before the targets and enclose in quotes `'` like elsewhere in this file: `'SPEC=$(SPEC)'`.